admission-controller/values.yaml

```diff
@@ -8,6 +8,9 @@ admission:
   tls_port: 8443
   cert_file_path: "/certs/admission.crt"
   cert_key_path: "/certs/admission.key"
+  excluded_resources:
+  -
+  -
```

Comment:
You might consider adding a `values.schema.json` to ensure that `excluded_resources` and other Helm chart values are well formed. It also helps to document the expected structure of the arguments (list vs hashmap, etc).

Reply:
Great suggestion! Added this file.